tasks/tag-and-push.yml

```diff
@@ -13,11 +13,7 @@
     force: true
 
 - name: Apply additional tag if there's more than one.
-  docker_image:
-    name: "{{ ecr_image_name }}:{{ ecr_image_tags.0 }}"
-    repository: "{{ ecr_image_name }}:{{ image_tag }}"
-    state: present
-    pull: false
+  command: "docker tag {{ ecr_image_name }}:{{ ecr_image_tags.0 }} {{ ecr_image_name }}:{{ image_tag }}"
```

Comment:
I think we need to tag the repo in the the tag as well (so like `{{ ecr_url }}/{{ ecr_image_name }}:{{ image_tag }}`).